### ISSUE TITLE
Can’t send Sats to my LNbits wallet

### ISSUE THREAD

**Issue by n-e-t-d-i-v-e-r** [lnbits/lnbits-legend#607]
Hi there

Don’t know if the issue is on my side or not. But I installed LNbits on my Raspibolt and everything went well so far. But if I’ll send Sats to an LNURLp-address into my wallet i get following error. Can anybody help me?
<img width="507" alt="Screenshot 2022-04-24 at 12 44 47" src="https://user-images.githubusercontent.com/87816768/165231409-60562cdb-7231-40f1-a7f8-58462dd71268.png">
![IMG_7498](https://user-images.githubusercontent.com/87816768/165231686-bacb8236-a306-4ee6-bf6f-ca59361b5482.jpg)

**Comment by arcbtc** [comment]
Where is the error?

**Comment by n-e-t-d-i-v-e-r** [comment]
![IMG_7497](https://user-images.githubusercontent.com/87816768/165510151-5bbb403c-b36d-4947-a5a2-55cacebbc82f.PNG)

**Comment by bezysoftware** [comment]
Looks like you're running your LNBits behind Tor where WoS cannot access it. You would need to expose it via HTTPS

**Comment by n-e-t-d-i-v-e-r** [comment]
@bezysoftware 
yes, it looks like, but to be honest i have no plan how to expose LNURLp via HTTPS. it’s strange but I can access LNbits behind tor at the moment and send sats to my LNbits wallet but not to LNURLp. maybe the title of this «issue» is wrong. should be something like: «I can access LNbits wallet behind tor but not LNURLp ...» :O)

**Comment by n-e-t-d-i-v-e-r** [comment]
meanwhile I found the reason why LNbits is reachable from outside and LNURLp not. It’s in the /etc/nginx/streams-enabled/lnbits-reverse-proxy.conf:

`upstream lnbits {
  server 127.0.0.1:5000;
}
server {
  listen 4002 ssl;
  proxy_pass lnbits;
}`

so my question now is. is there any related file for LNURLp, where I can insert such a server rule? and if yes, where will I find this and how should look the code to paste in ...?

**Comment by callebtc** [comment]
This is an nginx proxy issue, not an LNbits issue, correct?

**Comment by n-e-t-d-i-v-e-r** [comment]
not sure. from my point of view it is a somewhere wrong config. can’t say if it’s on LNURLp or on nginx side. fact is, with the above pasted lnbits-reverse-proxy.conf configuration LNbits works behind tor but LNURLp not and I got this configuration from the LNbits installation manual.
so maybe it’s a LNURLp issue and not an LNbits issue. if that’s so, am I wrong here?
